fs/install-to-emmc-begin

```diff
@@ -6,10 +6,12 @@ set -e
 
 echo "Installing Cadmium"
 
-# we need internet, TODO: make this run on voidlinux
-nmcli device wifi rescan
-sleep 4 # wait for it to show up
-nmtui connect
+# we need internet, TODO: make this run on voidlinux and arch
+#nmcli device wifi rescan
+#sleep 4 # wait for it to show up
+#nmtui connect
+
+read -p "Please use Ethernet, idk how the heck to get internet to work LOL"
```

Comment:
make it work  and remove this patch

Reply:
I didn't implement instal-to-emmc-begin. I am only installing using install-to-here. Maybe @FlameKat53 will implement EMMC?